lib/matplotlib/gridspec.py

```diff
@@ -196,6 +196,21 @@ def __init__(self, nrows, ncols, figure=None,
         ncols : int
             Number or columns in grid.
 
+        figure : ~.figure.Figure, optional
+
+        left, right, top, bottom : float
+            Space on each side of the subplots as a fraction of figure width.
```

Comment:
Ooops, sorry, so these are extents not spaces.  So maybe "Extent of the area where subplots will be placed as fraction of figure width and height; left cannot..."